install_openvslam.sh

```diff
@@ -34,12 +35,14 @@ echo ""
 export MAIN_DIR=$(pwd)
 
 #Begin openvslam dependencies
-mkdir lib_openvslam
+mkdir -p lib_openvslam
 cd lib_openvslam
 
+rm -rf openvslam
 git clone https://github.com/xdspacelab/openvslam.git
 
-mkdir external_dependencies
+rm -rf external_dependencies
```

Comment:
This causes a bit of an issue, as assuming only 1 library was not built correctly, this would cause you to reinstall and rebuild all the others. Similarly, if only a single file from openvslam failed, all the files would have to be rebuilt. Hence the preference for not removing the folder.

Reply:
True, but `mkdir` on an existing directory throws an error.  Maybe the better way is to check if the dir exists and if not, create it?